rules-emerging-threats/2023/Malware/SNAKE/proc_creation_win_malware_snake_service_execution.yml

```diff
@@ -0,0 +1,22 @@
+title: Potential SNAKE Malware Persistence Service Execution
+id: f7536642-4a08-4dd9-b6d5-c3286d8975ed
+status: experimental
+description: Detects specific child/parent process relationship indicative of a "WerFault" process running from the "WinSxS" as a service. This could be indicative of potential SNAKE malware activity as reported by CISA.
```

Comment:
```suggestion
description: Detects a specific child/parent process relationship indicative of a "WerFault" process running from the "WinSxS" as a service. This could be indicative of potential SNAKE malware activity as reported by CISA.
```